packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -44,6 +44,14 @@ import { IInterval, IntervalConflictResolver, IntervalTree, IntervalNode } from
 
 const reservedIntervalIdKey = "intervalId";
 
+export const IntervalOpType = {
+	ADD: "add",
+	DELETE: "delete",
+	CHANGE: "change",
+	PROPERTYCHANGE: "propertyChange",
```

Comment:
Should we have the opType name and the event name/revertibles functions match? I don't really have an opinion on which one we pick (propertyChange vs propertyChanged), but I do think they should be consistent.

Reply:
Good point. I modified "propertyChanged" to "propertyChange" to match the conjugation of "change", and I was intending to fix up the function names in a subsequent PR. But the event names are already inconsistent: "changeInterval" and "propertyChanged" (rather than "changeProperty"). I guess it would be best to be consistent with the existing event names.